client/src/components/CourseSearchBar.tsx

```diff
@@ -12,13 +12,14 @@ export const CourseSearchBar = ({
   handleInputChange,
 }: CourseSearchBarProps) => {
   const [searchSelected, setSearchSelected] = useState(false);
+  const parser = new DOMParser();
 
   return (
     <div className='relative'>
       <div className='relative w-full mt-4'>
         <div className='absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none'>
           <Search
-            size={20}
+            size={19}
```

Comment:
I think size 20 is fine, i meant the literal size of the circle in the icon was large -- we can leave this for now